### ISSUE TITLE
No support for rule type in .scesim files

### ISSUE THREAD

**Issue by sainath13** [apache/incubator-kie-tools#1839]
Hi Team,
I am trying to create new rules in .scesim files. The option is available but by default it is disabled. Will the support for RULE be launched ?

Reference image below:
<img width="611" alt="Screenshot 2023-07-06 at 4 55 18 PM" src="https://github.com/kiegroup/kie-tools/assets/6008957/acdadcdc-b1d1-4f9b-bcdc-990774c4b0f2">

**Comment by yesamer** [comment]
Hi @sainath13, yes, at this time we don't support RULE-based scesim files in kie-tools. Our intention is to implement it in the future, but we don't have an ETA to share.

**Comment by shreyasht** [comment]
This is a much-required feature as business central support has ended since drools 8.

**Comment by gomugomu0034** [comment]
Agreed, a lot of business processes will be affected without this support

**Comment by guptaniraj41** [comment]
+1 Much needed.. @yesamer can you provide a head start if possible, I would like to help in implementing the solution.

**Comment by tiagobento** [comment]
We're currently undergoing a full remodel of the Test scenario Editor. Yeser is leading this initiative, and we want to lay down the foundation to support DRL as well, but we're going to be starting from DMN, to match what we currently have. We can't share an ETA, as there's a lot of exploratory and research going on in this effort. 

Although I don't think the timing is appropriate for a code contributions, we could use this time to design the new DRL support together. What do you link and dislike about the current Test scenario Editor? I think any information you can contribute will be helpful for us design a better experience moving forward.

Thanks a lot for your interest!

**Comment by shreyasht** [comment]
@yesamer do we have an ETA for this feature?

**Comment by yesamer** [comment]
Hi @shreyasht, 
In the last couple of months, we started the scesim-editor refactoring initiative, you can find more details in this ticket -> https://github.com/apache/incubator-kie-tools/issues/3813 
The initiative's scope is to create a new scesim-editor based on a modern technology stack (React + TS).
The first version of the new editor will provide limited support of RULE-based scesim, but our goal is to evolve that to fully restore it.
We don't have a precise ETA for the first version and its future evolution, you can track the progress relying on the ticket I previously posted.
